content/applications/services/field_service/creating_tasks.rst

```diff
@@ -0,0 +1,80 @@
+============================
+Creating Field Service Tasks
+============================
+
+Creating tasks in **Field Service** from a sales order or a helpdesk ticket is faster and easier
+since these already contain a lot of information that will be imported automatically into your field
+service task, but it is also possible to create one manually from scratch.
+
+Manually
+========
+
+ #. Go to :guilabel:`Field Service`
+ #. Click on :guilabel:`New`
+ #. Fill in the :guilabel:`Task Title` and :guilabel:`Customer` (all other fields are optional),
+    then click :guilabel:`Save`
+
+Task configuration
```

Comment:
This part is actually already described in the Project doc so I think it would be best to just refer to that doc like I suggested above, and remove all of this so we don't have duplicate content. From what I can see, only the Worksheet template field is field service-specific but it's described in the Worksheets doc so we're okay. I was also wondering about Under warranty, but 1) I can't find it and 2) there's a good chance it would be sales-related as well